### PR TITLE
Fix Ignition 4-6 only

### PR DESCRIPTION
**Pull request by bobfuru**
Link to the Ignition docs site, not directly to the GitHub repo. Spec v3.1.0. Improve some wording. Fix capitalization of Ignition. Relates to https://github.com/openshift/openshift-docs/pull/29342.

### CODE REVIEW

**Comment by bobfuru** [comment]
@bgilbert PTAL
Cc: @miabbott

File: modules/machine-config-overview.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ a machine config that is injected directly into the {product-title} installer pr
 
 * MCO is only supported for writing to files in `/etc` and `/var` directories, although there are symbolic links to some directories that can be writeable by being symbolically linked to one of those areas. The `/opt` directory is an example.
 
-* Ignition is the configuration format used in MachineConfigs. See the link:https://github.com/coreos/ignition/blob/master/docs/configuration-v3_1.md[Ignition Configuration Specification v3.1.0] for details.
+* Ignition is the configuration format used in MachineConfigs. See the link:https://coreos.github.io/ignition/configuration-v3_1/[Ignition Configuration Specification v3.2.0] for details.
```
Comment (miabbott) [inline modules/machine-config-overview.adoc:24]: `Specification v3.1.0`

File: post_installation_configuration/machine-configuration-tasks.adoc [hunk 2]
```diff
@@ -24,7 +24,7 @@ Tasks in this section let you create `MachineConfig` objects to modify files, sy
...
 
-{product-title} version 4.6 supports link:https://github.com/coreos/ignition/blob/master/docs/configuration-v3_1.md[Ignition specification version 3.1].  All new machine configs you create going forward should be based on Ignition specification version 3.1.  If you are upgrading your {product-title} cluster, any existing Ignition specification version 2.x machine configs will be translated automatically to specification version 3.1.
+{product-title} version 4.6 supports link:https://coreos.github.io/ignition/configuration-v3_2/[Ignition specification version 3.2].  All new machine configs you create going forward should be based on Ignition specification version 3.2.  If you are upgrading your {product-title} cluster, any existing Ignition specification version 2.x machine configs will be translated automatically to specification version 3.2.
```
Comment (miabbott) [inline post_installation_configuration/machine-configuration-tasks.adoc:27]: Change any `3.2` refs to `3.1`

File: modules/machine-config-overview.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/machine-config-overview.adoc]: ah, my bad. resolved, thanks!

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

**Review by bgilbert — APPROVED** [review]
LGTM

**Comment by adellape** [comment]
:+1:

**Pull request merged**